libsrc/_DEVELOPMENT/target/cpm/library/cpm_sdcc_iy.lst

```diff
@@ -8,7 +8,7 @@
 @fcntl/fcntl_sdcc_iy.lst
 @font/font_4x8/fonts.lst
 @font/font_8x8/fonts.lst
-@font/fzx/fzx_sdcc_iy.lst
+@font/fzx/zx/zx_sdcc_iy.lst
```

Comment:
FZX is a ZX only library so I don't think these files are actually needed.

Reply:
Actually, given that it's CP/M I'm not sure there's much point in the other fonts either since there's no graphics functionality in newly

Reply:
I don't know, but there is inconsistency between the scz80 target and zsdcc_ix target paths for this library, and the zsdcc_iy target path. One of them must be right, and the other is wrong.

Reply:
Then, remove them all for all the cp/m targets seems like the relevant solution?
For this PR, that won't any effect, as only the referring `.lst` file is included in the core of the PR.